packages/react-native/ReactAndroid/src/main/java/com/facebook/react/config/EdgeToEdgeFlags.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.config
+
+/**
+ * A singleton object that holds flags related to edge-to-edge UI mode.
+ * This does not enable or apply edge-to-edge behavior, it simply tracks
+ * whether it has been flagged as enabled elsewhere in the application.
+ */
+public object EdgeToEdgeFlags {
+
+  /**
+   * Whether edge-to-edge mode has been flagged as enabled
+   */
+  @JvmStatic
+  public var isEnabled: Boolean = false
+}
```

Comment:
Instead of creating a new object, can we have this inside `ViewUtil`, `WindowUtil` or any other `*Util` class that we already have?